addons/mail/static/src/rtc/rtc_session_model.js

```diff
@@ -84,6 +85,18 @@ export class RtcSession {
         this.localVolume = value;
     }
 
+    async playAudio() {
+        if (!this.audioElement) {
+            return;
+        }
+        try {
+            await this.audioElement?.play?.();
+            this.audioError = undefined;
```

Comment:
`this.audioElement` already guarded above. Either remove first `?.` or just remove the `!this.audioElement` preemptive return.